community-extra-vars.yml

```diff
@@ -2,22 +2,22 @@
 acs:
   artifact_name: alfresco-content-services-community-distribution
   repository: "{{ nexus_repository.releases }}"
-  version: 7.2.1
+  version: 7.3.0
   edition: Community
 
 amps:
   aos_module:
     repository: "{{ nexus_repository.releases }}/aos-module/alfresco-aos-module"
-    version: 1.4.1
+    version: 1.5.0
   device_sync:
```

Comment:
I'm not sure why we have that here. Sync Service is only available to enterprise customers. Sync do no work with community.

Reply:
was introduced in https://github.com/Alfresco/alfresco-ansible-deployment/pull/431 by mistake I guess, I can try to fix it here